internal/service/project/resource_project.go

```diff
@@ -75,6 +76,7 @@ type TFProjectRSModel struct {
 	IsExtendedStorageSizesEnabled               types.Bool   `tfsdk:"is_extended_storage_sizes_enabled"`
 	IsCollectDatabaseSpecificsStatisticsEnabled types.Bool   `tfsdk:"is_collect_database_specifics_statistics_enabled"`
 	WithDefaultAlertsSettings                   types.Bool   `tfsdk:"with_default_alerts_settings"`
+	IsSlowOperationThresholdingEnabled          types.Bool   `tfsdk:"is_slow_operation_thresholding_enabled"`
```

Comment:
I'm following the naming of the API, but maybe `is_slow_operation_threshold_enabled` (removing `ing`) better?

Reply:
I'd prefer to align with the API